internal/tiger/config/config.go

```diff
@@ -357,6 +369,59 @@ func (c *Config) updateField(key string, value any) (any, error) {
 			return nil, fmt.Errorf("debug must be string or bool, got %T", value)
 		}
 
+	case "releases_url":
+		s, ok := value.(string)
+		if !ok {
+			return nil, fmt.Errorf("releases_url must be string, got %T", value)
+		}
+		c.ReleasesURL = s
+		validated = s
+
+	case "version_check_interval":
+		switch v := value.(type) {
+		case time.Duration:
+			if v < 0 {
+				return nil, fmt.Errorf("version_check_interval must be non-negative (0 to disable)")
+			}
+			c.VersionCheckInterval = v
+			validated = v
+		case string:
+			// Parse duration string like "1h", "30m", "24h"
+			d, err := time.ParseDuration(v)
+			if err != nil {
+				return nil, fmt.Errorf("invalid version_check_interval value: %s (must be a duration like '1h', '30m', etc.)", v)
+			}
+			if d < 0 {
+				return nil, fmt.Errorf("version_check_interval must be non-negative (0 to disable)")
+			}
+			c.VersionCheckInterval = d
+			validated = d
+		default:
+			return nil, fmt.Errorf("version_check_interval must be string or duration, got %T", value)
+		}
+
+	case "version_check_last_time":
+		switch v := value.(type) {
+		case time.Time:
+			c.VersionCheckLastTime = v
+			validated = v
+		case string:
+			// Try parsing as RFC3339 first, then as unix timestamp
+			t, err := time.Parse(time.RFC3339, v)
+			if err != nil {
+				// Try parsing as unix timestamp
+				i, err := strconv.ParseInt(v, 10, 64)
+				if err != nil {
+					return nil, fmt.Errorf("invalid version_check_last_time value: %s (must be RFC3339 timestamp or unix timestamp)", v)
+				}
+				t = time.Unix(i, 0)
+			}
+			c.VersionCheckLastTime = t
```

Comment:
I would add a sanity check that it's before now() or before now() and a day

Reply:
https://github.com/timescale/tiger-cli/pull/54/commits/cd05cc90d0acb4621c4f807f466915681631ad28